commit-police.js

```diff
@@ -107,7 +107,7 @@
     function doesNotEndWithDot(message) {
         // must necessarily end with an alpha-numeric value
         // or a new line... thanks, Bitbucket!@#$
-        return /[a-z0-9\n\s]$/i.test(message);
+        return /[a-z0-9]$/i.test(message.replace(/(\s+|\n+)$/, ''));
```

Comment:
this makes the test pass if you have 2 or more spaces at the end of the commit message subject...
It's better than nothing, so I'll merge it, but should be refactored soon!